### DISCUSSION TITLE
indefinite loading

### DISCUSSION THREAD

**Discussion by elliotcathcart** [dobladov/youtube2Anki#38] [Q&A]
Hi,

I just installed the extension, when I click on it it says "loading" and nothing happens. Any idea why this might be?

![image](https://user-images.githubusercontent.com/55717501/210158651-15bd19ae-3512-4cd3-b650-c22ce3132fbc.png)

**Comment by dobladov** [comment]
Can you right-click on the loading and select "Inspect"? here you can see a console that can give more information.

Please share that information so I can knwo what the problem might be.

Thanks.
